test/e2e/e2e.go

```diff
@@ -172,4 +174,105 @@ var _ = ginkgo.Describe("[efs-csi] EFS CSI", func() {
 	ginkgo.Context(testsuites.GetDriverNameWithFeatureTags(driver), func() {
 		testsuites.DefineTestSuite(driver, csiTestSuites)
 	})
+
+	f := framework.NewDefaultFramework("efs")
+
+	ginkgo.Context(testsuites.GetDriverNameWithFeatureTags(driver), func() {
+		ginkgo.It("should mount different paths on same volume on same node", func() {
+			ginkgo.By(fmt.Sprintf("Creating efs pvc & pv with no subpath"))
+			pvcRoot, pvRoot, err := createEFSPVCPV(f.ClientSet, f.Namespace.Name, f.Namespace.Name+"-root", "/")
+			framework.ExpectNoError(err, "creating efs pvc & pv with no subpath")
+			defer func() { _ = f.ClientSet.CoreV1().PersistentVolumes().Delete(pvRoot.Name, &metav1.DeleteOptions{}) }()
```

Comment:
Are PVC and Pods getting cleaned up automatically?

Reply:
yes, since namespaces gets cleaned up automatically